actors/builtin/multisig/multisig_actor.go

```diff
@@ -60,7 +60,7 @@ func (a *MultiSigActor) State(rt Runtime) (vmr.ActorStateHandle, MultiSigActorSt
 }
 
 type ConstructorParams struct {
-	AuthorizedParties     autil.ActorIDSetHAMT
+	AuthorizedParties     []abi.ActorID
```

Comment:
We should consistently use either actor IDs or addresses in AuthorizedParties here and the fields in Add/RemoveAuthorizedParty (and then avoid the Id/address conversions)